db_classes1.py

```diff
@@ -0,0 +1,156 @@
+import sqlite3
+
+
+class Database:
+    def __init__(self, db_name):
+        self.conn = sqlite3.connect(f'{db_name}.db')
+        self.c = self.conn.cursor()
+
+    def save(self, sql_script):
+        self.execute_sql(sql_script)
+        self.conn.commit()
+
+    def get(self, sql_script):
+        result = self.execute_sql(sql_script).fetchall()
+        self.conn.commit()
+        return result
+
+    def execute_sql(self, sql_script):
+        return self.c.execute(f'''{sql_script}''')
+
+
+class Department:
+    def __init__(self, ID, name, abbreviation, max_amount, database):
+        self.ID = ID
```

Comment:
Capital letters are used for constants. Correct name is `self.id`. The same for incoming parameters.